render/detailed_node.go

```diff
@@ -95,49 +110,35 @@ func MakeDetailedNode(r report.Report, n RenderableNode) DetailedNode {
 		}
 	}
 
-	tables := tables{}
+	// Connections table
 	{
-		rows := []Row{}
+		cRows := []Row{}
```

Comment:
why the variable rename?  We try and keep things like this out of PRs that add features.